makefile

```diff
@@ -187,6 +187,13 @@ $(buildDir)/go-test-config:cmd/go-test-config/make-config.go
 	GOPATH=$(shell dirname $(shell pwd)) $(gobin) build -o $@ $<
 #end generated config
 
+# generate rest model
+generate-rest-model:$(buildDir)/generate-rest-model
```

Comment:
So in terms of using this, is it just make generate-rest-model? Does the user always put their to-be-generated-struct in rest/model/schema/type_mapping.yml then?

Reply:
Correct, my intent is that all the generated stuff will be in a single file. So rest/model/schema/type_mapping.yml will bea list of all of our generated models